services/ownerService.js

```diff
@@ -23,7 +24,7 @@ exports.addRoom = async (req) => {
     description: description,
   }
 
-  await Hotel.updateOne({ $push: { rooms: room } })
+  await Hotel.updateOne({ _id: hotelId }, { $push: { rooms: room } })
```

Comment:
You can just do the same. req.params.hotelId
Or try:
Just await hotel.updateOne. i don't know if it Works but try it

Reply:
Or just hotel.rooms.push()